src/cobra/test/test_flux_analysis/test_loopless.py

```diff
@@ -37,14 +33,14 @@ def construct_ll_test_model():
     scope="function",
     params=[s for s in ["glpk", "cplex", "gurobi"] if s in sutil.solvers],
 )
-def ll_test_model(request):
+def ll_test_model(request: str) -> Model:
     """Return test model set with different solvers."""
     test_model = construct_ll_test_model()
     test_model.solver = request.param
     return test_model
 
 
-def test_loopless_benchmark_before(benchmark):
+def test_loopless_benchmark_before(benchmark) -> None:
```

Comment:
It doesn't improve much but you could annotate `benchmark` as `typing.Callable`.

Reply:
I'll add it.